src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -354,6 +355,27 @@ public function iProceededWithShippingMethod(ShippingMethodInterface $shippingMe
         $this->ordersClient->executeCustomRequest($request);
     }
 
+
+    /**
+     * @When /^I try to select inexistent "([^"]*)" payment method$/
+     */
+    public function iTryToSelectInexistentPaymentMethod(string $code): void
+    {
+        $paymentMethod = new PaymentMethod();
```

Comment:
Why do we need this variable?

Reply:
With this I could easily extract IRI, and I could add code that behat test is adding here. See line 374. Otherwise i thought about solution to concat strings but this could lead to changing code in many places (if for some reason we would change IRI or something). This way lets left generating IRI to our friendly functions from api platform :D